rocAL/include/parameters/parameter.h

```diff
@@ -33,6 +33,10 @@ class Parameter {
     /// used to internally renew state of the parameter if needed (for random parameters)
     virtual void renew(){};
 
+    virtual void create_array(unsigned batch_size){};
```

Comment:
maintain space between ) {

Reply:
The formatting is done by vscode formatter. The func definition is similar to renew() func in L34